practical/bash/import_as_create_annotation.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#   Copyright (C) 2018 University of Dundee. All rights reserved.
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License along
+#   with this program; if not, write to the Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+#
+# In this script we:
+#   * log in as another user
+#   * create a new dataset
+#   * import an image
+#   * upload a attachment
+#
+#
+dir=$(pwd -P)
+OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+PASSWORD=${PASSWORD:-ome}
+HOST=${HOST:-outreach.openmicroscopy.org}
+SUDOER=${SUDOER:-trainer-1}
+OMEUSER=${OMEUSER:-user-1}
+IMAGEPATH=${IMAGEPATH:-/}
+ATTACHMENTPATH=${ATTACHMENTPATH:-/}
+
+deleteimage=false
+deletefile=false
+
+if [ "$IMAGEPATH"="/" ]; then
+    touch image_to_import.fake
+    IMAGEPATH=$dir/image_to_import.fake
+    deleteimage=true
+fi
+#printf $IMAGEPATH
+
+if [ "$ATTACHMENTPATH"="/" ]; then
+    touch file_to_upload.csv
+    ATTACHMENTPATH=$dir/file_to_upload.csv
+    deletefile=true
+fi
+
+# Log in as another user
+$OMEROPATH login --sudo ${SUDOER} -u $OMEUSER -s $HOST -w $PASSWORD
+
+# Create a dataset as the specified user
+dataset=`$OMEROPATH obj new Dataset name='Basel-workflow'`
+
+# Import the image in the newly created dataset
+result=`$OMEROPATH import -T $dataset $IMAGEPATH --output ids`
+
+# Retrieve the Image's id. It is returned as Image:123
+imageid=`cut -d':' -f2 <<< $result`
+printf 'imageid %s \n' "$id"
+
+# Upload a CSV and attach it to the Image
```

Comment:
I am not sure this paragraph does the ``attach it to the Image`` part.

I guess this is done two paragraphs below, at https://github.com/ome/training-scripts/pull/35/files#diff-54247341f66f1673ff5f0d8697e623abR75